packages/andromeda-finance/src/validator_staking.rs

```diff
@@ -21,6 +23,13 @@ pub enum ExecuteMsg {
         validator: Option<Addr>,
         recipient: Option<AndrAddr>,
     },
+    WithdrawFund {},
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct Unstaking {
```

Comment:
Simple rename for clarity:

```suggestion
pub struct UnstakingTokens {
```

Reply:
Renamed